roles/common/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+common_timezone: 'Etc/UTC'
+
+# encfs
+encfs_password: "{{ lookup('password', 'credentials/' + role + '/encfs_password' length=32') }}"
```

Comment:
Good question. The first assumption is that this only goes in jessie and
so there's some likelihood people will generate fresh passwords for
everything.

As for migration I think you can run the script once and then edit the
files in credentials/ that Ansible creates.

I did consider Vault. There's also a host of other alternatives too,
like Mozilla's [sops](https://github.com/mozilla/sops),
[credstash](https://docs.ansible.com/ansible/playbooks_lookups.html#the-credstash-lookup),
etc.

I avoiding shifting to Vault because I think that probably is a separate
discussion. No secrets are encrypted now and this commit doesn't change
it. (I'd be open to switching to Vault.)


On 04/22, Mike Ashley wrote:
> > @@ -0,0 +1,4 @@
> > +common_timezone: 'Etc/UTC'
> > +
> > +# encfs
> > +encfs_password: "{{ lookup('password', 'credentials/' + role + '/encfs_password' length=32') }}"
> 
> How do you envision migrating existing passwords?  Also, did you consider using Ansible Vault?  The ansible docs for `lookup` suggested that Vault may be the better option.
> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/sovereign/sovereign/pull/526/files/b1633ab01bcd86674b04ffa5e37a2a88262141f2..fb6c94625068742775850d1294430bf1fb4c837a#r60715029